recipe/meta.yaml

```diff
@@ -31,17 +31,6 @@ source:
   - url: https://github.com/onnx/onnx/archive/refs/tags/v1.14.1.tar.gz
     sha256: e296f8867951fa6e71417a18f2e550a730550f8829bd35e947b4df5e3e777aa1
     folder: thirdparty/onnx/onnx
-  - url: https://github.com/gflags/gflags/archive/refs/tags/v2.2.2.tar.gz
-    sha256: 34af2f15cf7367513b352bdcd2493ab14ce43692d2dcd9dfc499492966c64dcf
-    folder: thirdparty/gflags/gflags
-  - url: https://github.com/nlohmann/json/archive/refs/tags/v3.11.2.tar.gz
-    sha256: d69f9deb6a75e2580465c6c4c5111b89c4dc2fa94e3a85fcd2ffcd9a143d9273
-    folder: thirdparty/json/nlohmann_json
-  - url: https://github.com/madler/zlib/archive/refs/tags/v1.2.13.tar.gz
-    sha256: 1525952a0a567581792613a9723333d7f8cc20b87a81f920fb8bc7e3f2251428
-    folder: thirdparty/zlib/zlib
-
-
 
 build:
   number: 3
```

Comment:
```suggestion
  number: 4
```

Reply:
when we remove tests, we don't need to bump revision

Reply:
if you are 100% the hashes havent changed then sure. in that case the builds wont get uploaded.

but keep it easy, bump the build number and that will avoid the solvers flip flopping in case there is a change in the hash